.travis.yml

```diff
@@ -1,17 +1,8 @@
-env:
-  global:
-  - GIT_NAME="'Markdown autodeploy'"
-  - GIT_EMAIL="goc@opensciencegrid.org"
-  - GH_REF="git@github.com:opensciencegrid/docs.git"
 language: python
-before_script:
-- pip install mkdocs
-- pip install MarkdownHighlight
-- pip install pygments
-- pip install linkchecker
-- pip install requests==2.9.2 # https://github.com/wummel/linkchecker/issues/649
+install: pip install -r ci/pip-requirements.txt
 script:
-- if [ "${TRAVIS_PULL_REQUEST}" == "false" ] && [ "${TRAVIS_BRANCH}" == "master" ]; then openssl aes-256-cbc -K $encrypted_75e9e0f098a4_key -iv $encrypted_75e9e0f098a4_iv -in deploy-key.enc -out deploy-key -d && ./travis/run_on_non_pull_requests.sh; else ./travis/run_on_pull_requests.sh; fi
+- ./ci/verify
+- if [ "${TRAVIS_PULL_REQUEST}" == "false" ]; then ./ci/deploy; fi
```

Comment:
Hm.  This seems brittle.

Can you put the `${TRAVIS_PULL_REQUEST}` logic in `./ci/deploy`?  The less logic, the better.